src/main/java/core/basesyntax/db/FruitDao.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import java.util.Map;
+
+public interface FruitDao {
```

Comment:
It's a good practice to use more descriptive names for your methods. The method name `putFruits` could be improved to reflect its purpose better. Consider renaming it to something like `addFruit` or `updateFruitQuantity` to make it clear what the method is intended to do.